modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -52,6 +52,40 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 
 
 
+== Bootstrap End-of-Life {centos} 6 Clients
+
+{centos} 6 is now at end-of-life, and the ISO images provided in the client repository for this operating system are out of date.
+Bootstrapping new {centos} 6 clients using these packages will fail.
+This failure will not happen on {centos} 6 clients that are already installed and bootstrapped.
+
+If you need to bootstrap new {centos} 6 clients you can remove the old repositories, and create a new repository to provide the correct packages.
```

Comment:
"Remove and add"?

I'd say that your procedure describes "changing" the baseurl for the existing repositories :-)